site/content/contribute/getting-started/inactive-contributions.md

```diff
@@ -13,13 +13,15 @@ This process describes how inactive contributions are managed at Mattermost, ins
 
 2. After 20 days of inactivity, a contribution becomes inactive.
  - Community Coordinator asks the submitter if the team can help with questions. They acknowledge that after another 30 days of inactivity the contribution will be closed. They also add a `lifecycle/2:inactive` label to the contribution.
- - Note: Contributions should never become orphaned because of reviewers. The Coordinator will be responsible for receiving a response from the reviewers during the stale period, which may be that the maintainers aren't able to accept the contribution in its current form.
+{{< note >}}
+Contributions should never become orphaned because of reviewers. The Coordinator will be responsible for receiving a response from the reviewers during the stale period, which may be that the maintainers aren't able to accept the contribution in its current form.
+{{< /note >}}
 
 3. After 30 days of inactivity, a contribution becomes orphaned.
  - Community Coordinator notes that the contribution has been inactive for 60 days, thanks for the contribution and closes the contribution. They also add an `lifecycle/3:orphaned` label to the contribution, and adds an `Up For Grabs` label to the associated help wanted ticket, if appropriate.
 
-Special notes:
-
-1. If the contribution is inactive but shouldn't be closed, the Coordinator adds a `lifecycle/frozen` label to the contribution. An example of this is when a design decision is being discussed but no decision has been arrived at yet.
-2. Once the contribution reaches the `lifecycle/2:inactive` state, it is eligible to be assumed by another community member interested in working on the ticket.
-3. Invalid PRs may be closed immediately without advancing through this lifecycle, especially if the contributor is unresponsive.
+{{< note >}} 
```

Comment:
I'm not fully convinced that this should be wrapped in a note. To me this is more a "normal" section and less an additional note.

Reply:
Thanks @hanzei !!!! Yes, I agree. Do we rephrase the heading to something that reads: 'Exceptions'

Reply:
Incorporated teh changes suggested. Please take a look. 
![Note p](https://user-images.githubusercontent.com/37059749/61703741-dba47f80-ad5f-11e9-8d30-286f8508a60f.jpg)